nservicebus/transactional-session/index.md

```diff
@@ -51,17 +50,45 @@ Sending messages in an atomic manner is done through the `ITransactionalSession`
 
 snippet: sending-transactional-session
 
-The persistence specific database session is accessible via the `transactionalSession.SynchronizedStorageSession` property or via dependency injection. See the [persistence specific documentation](/nservicebus/transactional-session/persistences) for more details.
+The persistence-specific database session is accessible via the `transactionalSession.SynchronizedStorageSession` property or via dependency injection. See the [persistence-specific documentation](/nservicebus/transactional-session/persistences) for more details.
 
 Once all the operations that are part of the atomic request have been executed, the session should be committed:
 
 snippet: committing-transactional-session
 
-Disposing the transactional session without committing, will roll-back any changes that were made.
+Disposing the transactional session, without committing, will roll back any changes that were made.
+
+### Advanced
+
+#### Maximum commit duration
+
+The maximum commit duration defaults to `TimeSpan.FromSeconds(15)`.
+
+The maximum commit duration limits the amount of time the transaction can attempt to commit the changes atomically. The value can be configured when opening the session.
```

Comment:
```suggestion
The maximum commit duration limits the amount of time it can take for a transaction to commit the changes. The value can be configured when opening the session.
```